tools/lint

```diff
@@ -47,9 +47,12 @@ def run():
     parser.add_option('--full',
                       action='store_true',
                       help='Check some things we typically ignore')
-    parser.add_option('--pep8',
-                      action='store_true',
-                      help='Run the pep8 checker')
+    parser.add_option('--py',
+                      action="store_true",
+                      help='Don\'t run expensive Javascript checks')
+    parser.add_option('--js',
+                      action="store_true",
+                      help='Don\'t run expensive Python checks')
```

Comment:
FYI I told Robert on chat that I'd want CSS/HTML checks when I'm in `--js` mode, but I am not super tied to that, since 75% of the time I really am just doing pure JS stuff when I'm in JS.